migrations/versions/8297ea2c9645_data_migrate_teams_to_membership.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+"""data migrate checkin teams to membership
+
+Revision ID: 8297ea2c9645
+Revises: 8cb98498a659
+Create Date: 2019-10-23 19:42:48.165500
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '8297ea2c9645'
+down_revision = '8cb98498a659'
+
+from uuid import uuid4
+
+from alembic import op
+from sqlalchemy.sql import column, table
+from sqlalchemy_utils import UUIDType
+import sqlalchemy as sa  # NOQA
+
+from funnel.models.membership import MEMBERSHIP_RECORD_TYPE
```

Comment:
Don't import from models in a migration. Reproduce the values here. You don't need the LabeledEnum, just a simple class or dict with name and value.